src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,45 @@
+import { React, useState } from "react";
+import "./navbar.css";
+
+export function Navbar() {
+  const [isMenuShow, setIsMenuShow] = useState(false);
+  const showMenu = () => {
+    setIsMenuShow(true);
+  };
+  const closeMenu = () => {
+    setIsMenuShow(false);
+  };
+  return (
+    <nav className="navbar semibold-font-weight">
+      <div className="openMenu">
+        <i className="fa fa-bars" onClick={showMenu}></i>
+      </div>
+      <a className="nav-icon-link nav-link link-no-style hide-icon" href="">
```

Comment:
use Link tag

Reply:
added routes, link in the next PR. So not changing here.